internal/http/apitester_test.go

```diff
@@ -8,12 +8,32 @@ import (
 	"net/http/httptest"
 	"reflect"
 	"testing"
+	"time"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/go-retryablehttp"
 	"github.com/saucelabs/saucectl/internal/apitest"
 	"github.com/saucelabs/saucectl/internal/config"
 	"github.com/stretchr/testify/assert"
+	"golang.org/x/time/rate"
 )
 
+func createTestRetryableHTTPClient(t *testing.T) *retryablehttp.Client {
+	t.Helper()
+	return &retryablehttp.Client{
+		HTTPClient: &http.Client{
+			Timeout:   10 * time.Second,
+			Transport: &http.Transport{Proxy: http.ProxyFromEnvironment},
+		},
+		RetryWaitMin: 1 * time.Second,
+		RetryWaitMax: 1 * time.Second,
```

Comment:
We practically don't need to wait at all:
```suggestion
		RetryWaitMin: 0 * time.Second,
		RetryWaitMax: 0 * time.Second,
```